ScrubBarTests/ScrubBarTests.swift

```diff
@@ -669,3 +856,29 @@ class AnimatingMock: Animating {
 func == (lhs: AnimatingMock.AnimationConfiguration, rhs: AnimatingMock.AnimationConfiguration) -> Bool {
     return String(describing: lhs) == String(describing: rhs)
 }
+
+extension ScrubBarItem: Equatable {}
+
+extension ScrubBarItem {
+
+    static func empty() -> ScrubBarItem {
```

Comment:
@andris-zalitis 

> `ScrubBarItem` has reference semantics ...

So has `UIColor`, `NSUserDefaults` and others. It's not preferred to use a property.